src/main/java/org/reaktivity/nukleus/http2/internal/types/stream/Http2FrameFW.java

```diff
@@ -105,4 +102,28 @@ public String toString()
                 type(), payloadLength(), flags(), streamId());
     }
 
+    public static int payloadLength(DirectBuffer buffer, int offset)
+    {
+        int length = (buffer.getByte(offset + LENGTH_OFFSET) & 0xFF) << 16;
+        length += (buffer.getByte(offset + LENGTH_OFFSET + 1) & 0xFF) << 8;
+        length += buffer.getByte(offset + LENGTH_OFFSET + 2) & 0xFF;
+        return length;
+    }
+
+    static FrameType type(DirectBuffer buffer, int offset)
+    {
+        return FrameType.from(buffer.getByte(offset + TYPE_OFFSET));
+    }
+
+    static int streamId(DirectBuffer buffer, int offset)
+    {
+        return buffer.getInt(offset + STREAM_ID_OFFSET, BIG_ENDIAN) & 0x7F_FF_FF_FF;
+    }
+
+    static void putPayloadLength(MutableDirectBuffer buffer, int offset, int payloadLength)
+    {
+        buffer.putShort(offset + LENGTH_OFFSET, (short) ((payloadLength & 0x00_FF_FF_00) >>> 8), BIG_ENDIAN);
+        buffer.putByte(offset + LENGTH_OFFSET + 2, (byte) (payloadLength & 0x00_00_00_FF));
+    }
```

Comment:
Let's either use inheritance or just inline the static methods to match what would have been generated.
Mutator methods should also not be present on the `Http2FrameFW` readonly flyweight.